lib/liquid/standardfilters.rb

```diff
@@ -42,7 +42,7 @@ def escape_once(input)
     end
 
     def url_encode(input)
-      CGI.escape(input) rescue input
```

Comment:
We already removed the `rescue input` for the escape filter in https://github.com/Shopify/liquid/pull/611, so we may as well do it here as well for the same reasons.